Spec/RealtimeClientPresence.swift

```diff
@@ -195,40 +195,15 @@ class RealtimeClientPresence: QuickSpec {
                     }
 
                     waitUntil(timeout: testTimeout) { done in
-                        channel.presence.get { membersPage, error in
+                        channel.presence.get { members, error in
                             expect(error).to(beNil())
-
-                            let membersPage = membersPage!
-                            expect(membersPage.items).to(haveCount(100))
-
-                            let firstItem = membersPage.items.first! as! ARTPresenceMessage
-                            expect(firstItem.clientId).toNot(beNil())
-                            expect(firstItem.data as? NSObject).to(equal(expectedData))
-
-                            let lastItem = membersPage.items.last! as! ARTPresenceMessage
-                            expect(lastItem.clientId).toNot(beNil())
-                            expect(lastItem.data as? NSObject).to(equal(expectedData))
-
-                            expect(membersPage.hasNext).to(beTrue())
-                            expect(membersPage.isLast).to(beFalse())
-
-                            membersPage.next { nextPage, error in
-                                expect(error).to(beNil())
-                                let nextPage = nextPage!
-                                expect(nextPage.items).to(haveCount(50))
-
-                                let firstItem = nextPage.items.first! as! ARTPresenceMessage
-                                expect(firstItem.clientId).toNot(beNil())
-                                expect(firstItem.data as? NSObject).to(equal(expectedData))
-
-                                let lastItem = nextPage.items.last! as! ARTPresenceMessage
-                                expect(lastItem.clientId).toNot(beNil())
-                                expect(lastItem.data as? NSObject).to(equal(expectedData))
-
-                                expect(nextPage.hasNext).to(beFalse())
-                                expect(nextPage.isLast).to(beTrue())
-                                done()
-                            }
+                            expect(members).to(haveCount(150))
+                            expect(members!.first).to(beAnInstanceOf(ARTPresenceMessage))
+                            expect(members!.first!.clientId).to(contain("user"))
```

Comment:
Not sure if we have this anywhere else, but I would quickly build an array of clientId values and ensure when `uniqued` there are 150 of them.